bioblend/_tests/pytest_galaxy_test_wrapper.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+import os
+import sys
+
+try:
+    import pytest
```

Comment:
Seems a good plan, would you prefer to get this merged and do the pytest migration in a new PR?